src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -97,6 +157,12 @@ private static GpioDriver CreateWindows10GpioDriver()
         /// <inheritdoc/>
         protected internal override int PinCount => _internalDriver.PinCount;
 
+        /// <summary>
+        /// The Raspberry Pi does support extended pin modes, but the this may depend on the actual driver implementation.
+        /// </summary>
+        /// <remarks>This returns true on Linux, false on Windows</remarks>
+        protected internal override bool ExtendedPinModeSupported => _internalDriver.ExtendedPinModeSupported;

```

Comment:
This should be public as well, otherwise user can't check it

Reply:
actually you akready have AlternatePinModeSettingSupported, this should be removed in that case

Reply:
True, this was just forgotten.

Reply:
I'm still seeing this